pages/projects.tsx

```diff
@@ -1,43 +1,184 @@
-import { Box, Container, Flex, Heading, Text } from "@chakra-ui/layout";
+import { Box, Container, Flex, Heading, Text, Link, SimpleGrid, VStack, Tag, HStack } from "@chakra-ui/react"
+import { Contact } from "../components/contact"
+import { BaseLayout } from "../layouts/base"
+import { Metadata } from "../components/metadata"
 
-import { Contact } from "../components/contact";
-import { BaseLayout } from "../layouts/base";
-import { Metadata } from "../components/metadata";
+const getStatusColor = (status) => {
+  switch (status) {
+    case "Active":
+      return "green";
+    case "Development":
+      return "blue";
+    case "Staged":
+      return "purple";
+    case "In Progress":
+      return "orange";
+    default:
+      return "gray";
+  }
+};
+
+const projects = [
+  {
+    id: 1,
+    fields: { slug: "/courseup" },
+    frontmatter: {
+      title: "CourseUp",
+      description: "A website built to simplify the experience of searching courses and building timetables for the University of Victoria (UVic).",
+      status: "Active",
+      tech: ["TypeScript", "React", "Firebase"],
+      meetingTime: "Fridays at 6-7pm",
+      lastUpdate: "Fall 2023",
+      term: "Fall 2023"
+    }
+  },
+  {
+    id: 2,
+    fields: { slug: "/coopme" },
+    frontmatter: {
+      title: "CoopMe",
+      description: "A website built to simplify the experience of searching for co-op placements for students in British Columbia.",
+      status: "Active",
+      tech: ["Next.js", "TypeScript", "Tailwind", "Prisma", "PostgreSQL"],
+      meetingTime: "Tuesdays at 4:30-5:30 PM",
+      lastUpdate: "Spring 2024",
+      term: "Spring 2024"
+    }
+  },
+  {
+    id: 3,
+    fields: { slug: "/scholarship-finder" },
+    frontmatter: {
+      title: "Scholarship Finder",
+      description: "A web application that allows users to search for scholarships based on various filters such as type of scholarship, faculty, and gender.",
+      status: "Active",
+      tech: ["React", "Node.js", "MongoDB"],
+      meetingTime: "Undecided",
+      lastUpdate: "Fall 2023",
+      term: "Fall 2023"
+    }
+  },
+  {
+    id: 4,
+    fields: { slug: "/lockers" },
```

Comment:
Could we call this locker-registration? I think their full title is ESS Locker Registration

Reply:
@liamshatzel I just updated it, let me know if there's anything else!